persistence-model/src/main/scala/hmda/persistence/messages/events/institutions/FilingEvents.scala

```diff
@@ -0,0 +1,10 @@
+package hmda.persistence.messages.events.institutions
+
+import hmda.model.fi.Filing
+import hmda.persistence.messages.CommonMessages.Event
+
+object FilingEvents {
+  sealed trait SubmissionEvent extends Event
```

Comment:
`SubmissionEvents.scala` also declares a trait called `SubmissionEvent`. Why does it need to be declared twice (or is this a typo)? What is it for?